foxi/onnxifi.h

```diff
@@ -1607,6 +1613,9 @@ ONNXIFI_PUBLIC onnxStatus ONNXIFI_ABI
  * @param[out] graph - pointer to the opaque handle for the created ONNXIFI
  *                     graph. If the function fails, and this pointer is
  *                     non-NULL, the handle is initialized to NULL.
+ * @param[inout] deferredWeightReader - pointer to an opaque handle to populate
```

Comment:
why is this inout?

Reply:
Some of the method here is not constant. It will modified the underlying "workspace".